sql/bench_test.go

```diff
@@ -281,6 +334,22 @@ func BenchmarkUpdate_Postgres(b *testing.B) {
 	benchmarkPostgres(b, runBenchmarkUpdate)
 }
 
+func BenchmarkUpdateTxn_Cockroach(b *testing.B) {
+	benchmarkCockroach(b, runBenchmarkUpdateTxn)
+}
+
+func BenchmarkUpdateTxn_Postgres(b *testing.B) {
+	benchmarkPostgres(b, runBenchmarkUpdateTxn)
+}
+
+func BenchmarkUpdateTwoRowsTxn_Cockroach(b *testing.B) {
```

Comment:
In other benchmarks I've used numbers in the benchmark name instead of spelling them out: `s/Two/2/g`.